plugin/compile-scss.js

```diff
@@ -4,38 +4,16 @@ const path = Plugin.path;
 const fs = Plugin.fs;
 
 const compileSass = promisify(sass.render);
-let _includePaths = [];
 const rootDir = (process.env.PWD || process.cwd()) + "/";
 
+const config = _getConfig('scss-config.json');
+const _includePaths = Array.isArray(config['includePaths']) ? config['includePaths'] : [];
+
 Plugin.registerCompiler({
   extensions: ['scss', 'sass'],
   archMatching: 'web'
 }, () => new SassCompiler());
 
-const toPosixPath = function toPosixPath(p, partialPath) {
-  // Sometimes, you can have a path like \Users\IEUser on windows, and this
-  // actually means you want C:\Users\IEUser
-  if (p[0] === "\\" && (!partialPath)) {
-    p = process.env.SystemDrive + p;
-  }
-
-  p = p.replace(/\\/g, '/');
-  if (p[1] === ':' && !partialPath) {
-    // transform "C:/bla/bla" to "/c/bla/bla"
-    p = `/${p[0]}${p.slice(2)}`;
-  }
-
-  return p;
-};
-
-const convertToStandardPath = function convertToStandardPath(osPath, partialPath) {
-  if (process.platform === "win32") {
-    return toPosixPath(osPath, partialPath);
-  }
-
-  return osPath;
-}
```

Comment:
These weren't being used anywhere, and haven't been for awhile as far as I can tell. Just removed them to clean stuff up.